Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs

```diff
@@ -119,5 +135,20 @@ internal static void DeleteDirectory(SftpClient client, string dir)
         if (client.Exists(dir))
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/56824f42a881d74709954eeb239fb474ee075540/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs#L127
This can be a one-line if-statement.

https://github.com/FrendsPlatform/Frends.SFTP/blob/56824f42a881d74709954eeb239fb474ee075540/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs#L131
This can be a one-line else-statement.